examples/README.md

```diff
@@ -87,6 +91,12 @@ terraform_backend_defaults:
     impersonate_service_account: <<SERVICE_ACCOUNT>>
 ```
 
+It should be noted that "gcs" is used in the example above, but the configuration accepts anything that the backend accepts.
```

Comment:
I'd replace this with something like "All Terraform-supported backends are supported by the Toolkit." Specify the backend using `type` and its [configuration block](https://developer.hashicorp.com/terraform/language/settings/backends/configuration#using-a-backend-block) using `configuration`. 

For the [gcs](https://developer.hashicorp.com/terraform/language/settings/backends/gcs) backend, you must minimally supply the `bucket` configuration setting. The `prefix` setting is generated automatically as "blueprint_name/deployment_name/group_name" for each deployment group. This ensures uniqueness.

If you wish to specify a custom prefix, use a unique value for each group following this example:

```yaml
deployment_groups:
- group: example_group
  terraform_backend:
    type: gcs
    configuration:
      bucket: your-bucket
      prefix: your/object/prefix
```

Reply:
I think you phrased it nicely here so I replaced the section with your suggestion.